mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -303,9 +304,9 @@ private JobExecutionUserInfoCollection mapRowToJobExecutionUserInfoCollection(Ro
 
   private JobExecutionUserInfo mapRowToJobExecutionUserInfoDto(Row row) {
     JobExecutionUserInfo jobExecutionUserInfo = new JobExecutionUserInfo();
-    jobExecutionUserInfo.setUserId(row.getValue(USER_ID_FIELD).toString());
-    jobExecutionUserInfo.setJobUserFirstName(row.getValue(FIRST_NAME_FIELD).toString());
-    jobExecutionUserInfo.setJobUserLastName(row.getValue(LAST_NAME_FIELD).toString());
+    jobExecutionUserInfo.setUserId(row.getUUID(USER_ID_FIELD).toString());
+    jobExecutionUserInfo.setJobUserFirstName(StringUtils.defaultIfEmpty(row.getString(FIRST_NAME_FIELD), "No first name"));
```

Comment:
@Abdulkhakimov  Thank you for fixing this! Please change for an empty value here.

Reply:
Done!